modules/modules.nix

```diff
@@ -129,6 +129,7 @@ let
     (loadModule ./services/dunst.nix { })
     (loadModule ./services/dwm-status.nix { condition = hostPlatform.isLinux; })
     (loadModule ./services/emacs.nix { condition = hostPlatform.isLinux; })
+    (loadModule ./services/etesync-dav.nix { })
```

Comment:
Systemd is not available on non-Linux platforms so I would suggest adding a corresponding condition.